source/manual/data-gov-uk-architecture.html.md

```diff
@@ -16,14 +16,42 @@ review_in: 6 months
+[statistics]: http://statistics.data.gov.uk
+[land-registry]: http://landregistry.data.gov.uk
+[csw]: http://csw.data.gov.uk/geonetwork/srv/en/main.home
+[legislation]: http://legislation.data.gov.uk
+[location-mde]: http://locationmde.data.gov.uk
+[guidance]: http://guidance.data.gov.uk
+[business]: http://business.data.gov.uk/id/company/09747720
+[location]: http://location.data.gov.uk/registry/
+[environment]: http://environment.data.gov.uk/index.html
+[guidance-github]: https://github.com/datagovuk/guidance
+[open-data-policy]: https://www.gov.uk/government/publications/open-data-white-paper-unleashing-the-potential
+[inspire]: http://inspire.ec.europa.eu/about-inspire
+[uk-location-programme]: https://inspire.ec.europa.eu/events/conferences/inspire_2010/presentations/258_pdf_presentation.pdf
+[contract-finder]: https://data.gov.uk/data/contracts-finder-archive/
+[contract-finder-new]: https://www.contractsfinder.service.gov.uk/Search
+[land-registry-birth]: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/246732/0247.pdf
+[reference]: http://reference.data.gov.uk
+[time-interval-service]: https://github.com/alphagov/IntervalServer
+[heroku]: https://docs.publishing.service.gov.uk/manual/review-apps.html#use-the-shared-heroku-account
 
-The `data.gov.uk` platform is used to publish and view datasets. A dataset is a document about a collection of links to documentation or data hosted somewhere on the Internet. The platform has the following software.
+The `data.gov.uk` platform is used to publish and view datasets. A dataset is a document about a collection of links to documentation or data hosted somewhere on the Internet. The platform has the following services.
 
   * [CKAN] is the legacy publishing and finder app for datasets ('packages'). It also runs Nginx to support [Find].
   * [Find] is the public frontend for searching datasets using Elasticsearch. It replaces CKAN for certain routes.
   * [Publish] is the prototype publishing app for datasets. It currently syncs with CKAN to populate Elasitcsearch.
-  * *TODO: Other services (contract-finder-archive)*
-  * *TODO: Other services ([crime,education,finance,services,source,transport,weather,reference,guidance].data.gov.uk)*
+  * [Statistics] is owned by the Office for National Statistics and was established as part of the [Open Data Policy][open-data-policy].
+  * [Environment][environment] is owned by DEFRA and was created with the [Location] service as part of the [Open Data Policy][open-data-policy].
+  * [Land Registry][land-registry] is owned by the same and [was created][land-registry-birth] to publish linked data as part of the [Open Data Policy][open-data-policy].
+  * [Catalog Service for the Web][csw] is owned by Ordnance Survey and serves [INSPIRE] datasets to the EU Geoportal.
+  * [Location Metadata Editor][location-mde] is owned by DEFRA and publishes [INSPIRE] location metadata e.g. to enrich datasets.
```

Comment:
Stylistically, "for example" would be preferable here – https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#eg-etc-and-ie